extensions/HuggingFaceTransformers/python/requirements.txt

```diff
@@ -5,11 +5,13 @@ pylint
 # AIConfig
 python-aiconfig
 
-#Hugging Face Transformers
-transformers
+#Hugging Face Libraries
+accelerate
```

Comment:
what is accelerate used for? I don't see it being used in the code

Reply:
It's used under the hood to help speed up image generation. It's not necessary but helps and is recommended when you run the text2image model without it